app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -80,6 +80,7 @@ def analytics_properties
         context: context,
         multi_factor_auth_method: params[:otp_delivery_preference],
         confirmation_for_add_phone: confirmation_for_add_phone?,
+        mfa_id: user_session[:phone_id],
```

Comment:
can we name this `mfa_phone_id`? Or prefix the ID with `phone_`? We name all these keys `mfa_id` but they're all from different sources, so we'll have ID collisions and may accidentally look up the wrong thing if we ever investigate

Reply:
The ids won't collide because they are in the context of another field `multi_factor_auth_method`.  Moreover, if these fields were in a table you wouldn't have a new column for each mfa id.

Reply:
Right, but I'm saying that I think it's not good to have to check two separate keys to figure out what we have. It means if we ever want to query how often an MFA method is used, we have to remember to search by two keys, `multi_factor_auth_method` as well as `mfa_id`

I think the least error-prone way to record all these IDs from different source tables to add string prefixes:

```suggestion
        mfa_id: "phone-#{user_session[:phone_id]}",
```